modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -0,0 +1,41 @@
+= Discover What’s Changed in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can see a history of changes you make to a project in Flow Designer.
+These changes are automatically saved every few minutes while you are working on your project.
+The saved states are listed by time/date and activity, and their name depends on the activity you were doing at the time the save occurred.
+For example, configuring a trigger in a flow.
+
+Being able to view a saved state of a project is useful if you want to:
+
+* Detect what is or is not working between two different versions. For example, you started with a working, tested flow but during the configuration, you see errors and you want to revert back to the working state.
+* Compare two versions of a flow. For example, you might want to publish different versions of the same flow.
+
+
+== Procedure
```

Comment:
"Procedure" seems a little overly formal to me.  Consider "How to use it".  Your call.

Reply:
I was following the structure and style that Robert had set up for the other topics. If we change it for this topic, it will stand out as being different from the others.

Reply:
Up to you